### ISSUE TITLE
'Failed at the node-webrender@1.0.8 install script...' error

### ISSUE THREAD

**Issue by papajo** [cztomsik/slack-app#1]
cloned the repo and ran npm i and got the error:
npm WARN slack-app@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-webrender@0.1.8 install: `neon build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-webrender@0.1.8 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

![image](https://user-images.githubusercontent.com/2457792/51063216-013e1880-15c8-11e9-8070-40ca07862e16.png)
[2019-01-09T17_56_14_690Z-debug.log](https://github.com/cztomsik/slack-app/files/2751368/2019-01-09T17_56_14_690Z-debug.log)

**Comment by cztomsik** [comment]
This is weird, I don't think you're missing dependency, can you provide exact commands you've done?

Also, some info about your platform would be great help:
- osx version (`uname -a`)
- rust version (`rustc --version`)
- cargo version (`cargo --version`)

BTW: are you sure you had rust in path when running `npm i`? If you first install rust you have to open new terminall IIIRC.

**Comment by papajo** [comment]
// ♥ uname -a
Darwin PJoshis-MacBook-Pro.local 18.2.0 Darwin Kernel Version 18.2.0: Mon Nov 12 20:24:46 PST 2018; root:xnu-4903.231.4~2/RELEASE_X86_64 x86_64
// ♥ rustc --version
-bash: rustc: command not found
// ♥ cargo --version
-bash: cargo: command not found
does this mean i have to install rust and cargo? If so how? Also can you create a slack channel to discuss these issues if possible?
Thx. PJ.

**Comment by papajo** [comment]
still the same issue after rust installation. (yes from a new shell window)
// ♥ rustc --version
rustc 1.31.1 (b6c32da9b 2018-12-18)

// ♥ cargo --version
cargo 1.31.0 (339d9f9c8 2018-11-16)

logs:
[2019-01-13T18_28_00_222Z-debug.log](https://github.com/cztomsik/slack-app/files/2753355/2019-01-13T18_28_00_222Z-debug.log)

**Comment by papajo** [comment]
Okay finally seems to have figured out. Refer to the following SA link:
[](https://apple.stackexchange.com/questions/254380/why-am-i-getting-an-invalid-active-developer-path-when-attempting-to-use-git-a)

tldr: $ xcode-select --install
to install developer tools and fix the issue. (I hope)

**Comment by papajo** [comment]
Solution: 
1. update to latest MacOS
2. run $ xcode-select --install
3. npm i
